app/controllers/settings/profile_controller.rb

```diff
@@ -1,23 +1,35 @@
 # frozen_string_literal: true
 
-class Settings::ProfileController < Sellers::BaseController
+class Settings::ProfileController < Settings::BaseController
   before_action :authorize
 
   def show
     @title = "Settings"
-    @profile_presenter = ProfilePresenter.new(pundit_user:, seller: current_seller)
-    @settings_presenter = SettingsPresenter.new(pundit_user:)
+    profile_presenter = ProfilePresenter.new(pundit_user:, seller: current_seller)
 
-    @react_component_props = @settings_presenter.profile_props.merge(
-      @profile_presenter.profile_settings_props(request:)
+    render inertia: "Settings/Profile/Show", props: settings_presenter.profile_props.merge(
+      profile_presenter.profile_settings_props(request:)
     )
   end
 
   def update
-    return render json: { success: false, error_message: "You have to confirm your email address before you can do that." } unless current_seller.confirmed?
+    is_inertia_request = request.headers["X-Inertia"].present?
```

Comment:
This method is used in the profile settings page and the user profile pages, we can just change the condition when we migrate the user profile pages